models/snapchat.yml

```diff
@@ -115,17 +115,17 @@ models:
       - name: campaign_name
         description: The name of the campaign in Snapchat.
       - name: currency
-        description: The current used by the account in Snapchat.
+        description: The currency used by the account in Snapchat.
       - name: spend
-        description: The spend on the ad in the given day.
+        description: The spend on the campaign in the given day.
       - name: impressions
-        description: The number of impressions the ad had on the given day.
+        description: The number of impressions the campaign had on the given day.
       - name: swipes
-        description: The number of swipes the ad had on the given day.
+        description: The number of swipes the campaign had on the given day.
       - name: conversion_purchases
```

Comment:
It looks like this field isn't brought into the end model, is this supposed to be brought in or should this be removed from the yml?